src/traces/scatter/constants.js

```diff
@@ -0,0 +1,14 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+module.exports = {
+    PTS_LINESONLY: 20
```

Comment:
slightly abusive, but it's the only way to clear us of circular dependencies while keeping things :palm_tree: